esbuildBabel.ts

```diff
@@ -57,7 +57,11 @@ export const esbuildPluginBabel = (options: ESBuildPluginBabelOptions = {}): Plu
 			});
 		};
 
-		build.onLoad({ filter, namespace }, async args => {
+		build.onLoad({ filter: /.*/, namespace }, async args => {
+      const shouldTransform = filter.test(args.path);
+
+      if (!shouldTransform) return;
+
```

Comment:
Please fix the formatting to match the original file's tab-based configuration

Reply:
Done in #23 . I could not amend this PR